### ISSUE TITLE
Submit to better discord theme library

### ISSUE THREAD

**Issue by neonvoidx** [eldritch-theme/betterdiscord#1]
Want to submit it to the BetterDiscord theme library, currently waiting on 2 day approval process

**Comment by neonvoidx** [comment]
Won't do. Betterdiscord doesn't allow recolors, you have to make an entire different theme appearance wise outside of color

So manual install is the only way, theme repo plugin doesn't really offer any shortcuts anyways
